docs/src/pages/customization/default-theme/DefaultTheme.js

```diff
@@ -78,7 +81,1685 @@ function DefaultTheme(props) {
 
 DefaultTheme.propTypes = {
   classes: PropTypes.object.isRequired,
-  theme: PropTypes.object.isRequired,
+  theme: PropTypes.shape({
```

Comment:
Not sure how to "fix" this, it's transpiled like this because I'm using the `Theme` type in `Props`, and that's correct, but now the javascript code is pretty much unreadable.

Reply:
You need to tell it to not resolve props with the name `theme`, same as what was done with `classes`
https://github.com/mui-org/material-ui/blob/0168a6d2ecb7c684dd12a7ea8e5cf210f6ee4944/docs/scripts/formattedTSDemos.js#L91-L93

Reply:
Thank you! That will be very useful in the future.